src/content/getting-started/_index.en.md

```diff
@@ -68,6 +68,9 @@ tested with the following network (CNI) Plugins:
 * [Calico](https://www.projectcalico.org/) (see the [Calico-specific deployment instructions](../operations/deployment/calico/))
 * [OVN](https://github.com/ovn-org/ovn-kubernetes)
 
+Submariner supports all currently-supported Kubernetes versions, which are the [three most recent
+releases](https://kubernetes.io/releases/).
```

Comment:
```suggestion
Submariner supports all currently-supported Kubernetes versions, as determined by [the Kubernetes release policy](https://kubernetes.io/releases/).
```

to make it explicit that we’re following K8s policy and to avoid having to change the text if the policy changes.